integration_tests/base_test.py

```diff
@@ -1,11 +1,12 @@
-# pylint: disable=no-member
+# pylint: disable=no-member, protected-access, attribute-defined-outside-init
```

Comment:
❤️ but also maybe still a code smell that indicates we can clean up the class hierarchy a bit.